x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/SearchableSnapshotAction.java

```diff
@@ -92,6 +91,11 @@ public String getSnapshotRepository() {
 
     @Override
     public List<Step> toSteps(Client client, String phase, StepKey nextStepKey) {
+        throw new UnsupportedOperationException();
```

Comment:
nit: `assert false` here too? For the record I'd rather just add the parameter to `toSteps()` everywhere rather than have this uncalled overload, i.e, the opposite of what we did in d4b2e32137b429275fc1372630fc33ce49bb3887.

Reply:
See 63816a07d4ab7802b2be8b0721d99d1b31df1c3d. I left the overload as is.